content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+
+- Define a high minimum for allocation at any scale
+- Can strand capacity by reserving more than is needed
+- Shift operational burden to adjust allocation if more than the default is needed
+
+The sidecar architecture forces this mode of allocation on users. Ambient solves these issues by leveraging a new architecture that separates the responsibilities of zero-trust networking and L7 policy handling. This is done with two new components to Istio: ztunnels and waypoint proxies.
+
+- [Ztunnels](/blog/2023/rust-based-ztunnel/) are a brand new Istio component written in Rust that are designed to be fast, secure, and lightweight. Ztunnels are deployed per node on a cluster and enable the most basic service mesh configurations for L4 features such as mTLS, telemetry, authentication, and L4 authorizations.
+- [Waypoint proxies](/blog/2023/waypoint-proxy-made-simple/) provide L7 mesh features such as VirtualService routing, L7 telemetry, and L7 authorizations policies. Waypoints are still based on Envoy and are deployed at the namespace level per ServiceAccount.
+
+These ztunnels and waypoint proxies work in tandem to replace sidecars in the Istio service mesh. So let’s take a closer look at how the two architectures compare in the tests above.
+
+## A Closer Look
+
+{{< image width="100%"
+    link="cpu-by-pod.png"
+    caption="CPU usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="memory-by-pod.png"
+    caption="Memory usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="resource-usage.png"
+    caption="CPU and Memory usage for cluster"
+    >}}
+
+Let’s start by looking at CPU usage by pod. In the sidecar scenarios, the containers utilizing the most CPU resources are the [frontend](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/src/frontend) service sidecars, which is main server hit by the load generators and also responsible for making any subsequent calls to other underlying microservices.
+
+In the ambient scenarios, a particular ztunnel's performance depends on the distribution of services across the nodes in the cluster. In the ambient with both ztunnel and waypoint proxy scenario, there is an increase in average CPU usage per ztunnel as traffic is now directed to waypoint proxies. The waypoint proxies themselves consume more resources than their sidecar counterparts since there are only three instances processing all traffic in the namespace versus one per pod in the sidecar scenario.
```

Comment:
* The waypoint proxies themselves consume more resources than their sidecar counterparts* 

I would say something like this is expected because waypoint proxy handles L7 processing for more than multiple application pods (in your case, i think it is 48 pods).